### DISCUSSION TITLE
Excel report

### DISCUSSION THREAD

**Discussion by marchoeppner** [bio-raum/FooDMe2#23] [General]
@gregdenay you mentioned that people usually use Excel tables to document their analyses. We could easily add something like that to the pipeline; but I am not 100% sure what sort of structure the document should have. Having one worksheet per sample seems impractical for larger data sets (70+ tabs, yeah). Something like:

| sample name | taxonomy assignments |
| sample A         | Bos taurus (0.9), Sus scrofa (0.1)

would be easy enough, although not exactly easy to "read" for complex samples.

**Comment by gregdenay** [comment]
I would just dump all result in a semi-long format 
```
| sample name | taxon | quantity |
```
or something similar and let users format it themselves in excel. 

Not sure we really need the hassle of producing an excel file too, it's easy enough to export tables form MultiQC

**Reply by marchoeppner** [reply]
Fair enought; it is pretty simple though - basically already done. Just need to figure out the best representation. I'll try your suggestion and we can tweak it as we go along.

**Reply by marchoeppner** [reply]
Should now be included - stores a table under reports/, named based on `--run_name` 

It's pretty basic right now, but maybe we find some additional uses ;)
